core/domain/exp_domain.py

```diff
@@ -2387,6 +2387,32 @@ def _convert_states_v31_dict_to_v32_dict(cls, states_dict):
 
         return states_dict
 
+    @classmethod
+    def _convert_states_v32_dict_to_v33_dict(cls, states_dict):
+        """Converts from version 32 to 33. Version 33 adds a new
+        customization arg to MultipleChoiceInput which allows
+        answer choices to be shuffled.
+
+        Args:
+            states_dict: dict. A dict where each key-value pair represents,
+                respectively, a state name and a dict used to initialize a
+                State domain object.
+
+        Returns:
+            dict. The converted states_dict.
+        """
+        for state_dict in states_dict.values():
+            if state_dict['interaction']['id'] == 'MultipleChoiceInput':
+                customization_args = state_dict[
+                    'interaction']['customization_args']
+                customization_args.update({
+                    'showChoicesInShuffledOrder': {
+                        'value': True
```

Comment:
@seanlip Not sure if we should the value to `True` for existing explorations. What if there are some explorations where the author doesn't want this, but doesn't see this automatic change?

Reply:
I have changed it to false. Done.